common-utils/src/main/java/eu/arrowhead/common/http/filter/authorization/ManagementServiceFilter.java

```diff
@@ -18,18 +24,70 @@
 @Order(Constants.REQUEST_FILTER_ORDER_AUTHORIZATION_MGMT_SERVICE)
 public class ManagementServiceFilter extends ArrowheadFilter {
 
-	// TODO: if an endpoint is management, then check sysop, whitelist or authorization (depending on management.policy config)
-	// requester comes from the request attribute "arrowhead.authenticated"
+	//=================================================================================================
+	// members
+
+	@Autowired
+	private SystemInfo sysInfo;
+
+	private static final String mgmtPath = "/mgmt/";
+
+	private final Logger logger = LogManager.getLogger(this.getClass());
 
 	//=================================================================================================
 	// assistant methods
 
 	//-------------------------------------------------------------------------------------------------
 	@Override
 	protected void doFilterInternal(final HttpServletRequest request, final HttpServletResponse response, final FilterChain chain) throws IOException, ServletException {
+		logger.debug("ManagementServiceFilter.doFilterInternal started...");
+
+		final String requestTarget = request.getRequestURL().toString();
+		if (requestTarget.contains(mgmtPath)) {
+			final String systemName = (String) request.getAttribute(Constants.HTTP_ATTR_ARROWHEAD_AUTHENTICATED_SYSTEM);
+			boolean allowed = false;
```

Comment:
we should normalize the systemName (lowercase, trim)

Reply:
Can't we be sure here that the authentication filter is already normalized it?

Reply:
we can do that, but in that case we have to implement it 6 times.